sw/device/tests/sram_ctrl_scrambled_access_test.c

```diff
@@ -2,14 +2,31 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
 #include "sw/device/lib/runtime/log.h"
 #include "sw/device/lib/testing/check.h"
 #include "sw/device/lib/testing/sram_ctrl_testutils.h"
 #include "sw/device/lib/testing/test_framework/test_main.h"
 #include "sw/device/lib/testing/test_framework/test_status.h"
 
+#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
+
 const test_config_t kTestConfig;
 
+static dif_sram_ctrl_t sram_ctrl;
+
+/**
+ * Retention SRAM start address (inclusive).
+ */
+static const uintptr_t kSramStartAddr =
```

Comment:
nit: can you mention that this is the retention SRAM in this constant somewhere? e.g. `kRetSramStartAddr`.

Reply:
Done.